firecrown/models/cluster/recipes/cluster_recipe.py

```diff
@@ -22,7 +21,7 @@ def __init__(self, parameter_prefix: None | str = None) -> None:
     @abstractmethod
     def evaluate_theory_prediction(
         self,
-        cluster_theory: ClusterAbundance,
+        cluster_theory_list: list,
```

Comment:
I had to change this for a list because the recipe for DeltaSigma needs both cluster abundance and cluster delta sigma, and this was giving a warning since the child method did not matched this abstract method.